crates/concensus/miner/Cargo.toml

```diff
@@ -30,7 +30,10 @@ log = "0.4"
 parity-runtime = { path = "../../runtime/runtime" }
 parking_lot = "0.7"
 price-info = { path = "./price-info", optional = true }
-rlp = { version = "0.3.0", features = ["ethereum"] }
+rlp = { version = "0.4.6" }
+serde = "1.0"
+serde_derive = "1.0"
```

Comment:
Afaik newer versions of serde have serde_derive built in and don’t need a separate import.

Reply:
`serde = { version = "1.0", features = ["derive"] }` this should probably work. https://serde.rs/derive.html

Reply:
Yes, that worked.